src/test/java/nearsoft/academy/bigdata/recommendation/MovieRecommender.java

```diff
@@ -0,0 +1,145 @@
+package nearsoft.academy.bigdata.recommendation;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.zip.GZIPInputStream;
+import org.apache.mahout.cf.taste.common.TasteException;
+import org.apache.mahout.cf.taste.impl.model.file.FileDataModel;
+import org.apache.mahout.cf.taste.impl.neighborhood.ThresholdUserNeighborhood;
+import org.apache.mahout.cf.taste.impl.recommender.GenericUserBasedRecommender;
+import org.apache.mahout.cf.taste.impl.similarity.PearsonCorrelationSimilarity;
+import org.apache.mahout.cf.taste.model.DataModel;
+import org.apache.mahout.cf.taste.recommender.RecommendedItem;
+import org.apache.mahout.cf.taste.neighborhood.UserNeighborhood;
+import org.apache.mahout.cf.taste.recommender.UserBasedRecommender;
+import org.apache.mahout.cf.taste.similarity.UserSimilarity;
+
+import com.google.common.collect.BiMap;
+import com.google.common.collect.HashBiMap;
+
+/**
+ *
+ * @author ifrescas
+ */
+
+public class MovieRecommender {
+    
+    private int totalReviews = 0;
+    private int totalProducts = 0;
+    private int totalUsers = 0;
+
+    //Hash map para almacenar ids
+    //HashMap<String, Integer> productsMap = new HashMap<String, Integer>();
+    //HashMap<String, Integer> usersMap = new HashMap<String, Integer>();
+
+    BiMap<String, Integer> productsMap = HashBiMap.create();
+    BiMap<String, Integer> usersMap = HashBiMap.create();
+
+
+    public MovieRecommender(String path) throws  IOException {
```

Comment:
This class has too many responsibilities on its own. When making new classes for the project, use Single Responsibility to model your classes. 

Reference: [wiki article on Solid](https://en.wikipedia.org/wiki/SOLID)